forui/lib/src/theme/style/zinc_style.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+
+import 'package:forui/forui.dart';
+
+/// Zinc style of [FThemeData].
+///
+/// The zinc style is based on [shadcn/ui](https://ui.shadcn.com/themes).
+extension FZincStyle on Never {
```

Comment:
Perhaps we can align the terminology used in Forui and rename this to `FZincThemeData`/`ZincThemeData`?